src/main/java/org/candlepin/service/exception/OrgForCloudAccountNotEntitledYetException.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2009 - 2023 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.service.exception;
+
+/**
+ * The OrgForCloudAccountNotCreatedYetException is used when organization does not exist for cloud account
+ */
+public class OrgForCloudAccountNotEntitledYetException extends RuntimeException {
```

Comment:
The comment needs updating to match what the exception is for

Reply:
done